src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -397,6 +442,45 @@ void CALLBACK cfApiCancelFetchPlaceHolders(const CF_CALLBACK_INFO *callbackInfo,
     qCDebug(lcCfApiWrapper) << "Cancel fetch placeholder:" << path << requestId;
 }
 
+void CALLBACK cfApiFetchPlaceHolders(const CF_CALLBACK_INFO *callbackInfo, const CF_CALLBACK_PARAMETERS *callbackParameters)
+{
+    qDebug(lcCfApiWrapper) << "Fetch data callback called. File size:" << callbackInfo->FileSize.QuadPart;
+    qDebug(lcCfApiWrapper) << "Desktop client proccess id:" << QCoreApplication::applicationPid();
+    qDebug(lcCfApiWrapper) << "Fetch data requested by proccess id:" << callbackInfo->ProcessInfo->ProcessId;
+    qDebug(lcCfApiWrapper) << "Fetch data requested by application id:" << QString(QString::fromWCharArray(callbackInfo->ProcessInfo->ApplicationId));
```

Comment:
would be interesting to see what [`callbackParameters->FetchPlaceholders.Pattern`](https://learn.microsoft.com/en-us/windows/win32/api/cfapi/ns-cfapi-cf_callback_parameters#:~:text=FetchPlaceholders.Pattern) is set to as well

not sure if we actually need to do something with that though, as the docs for that also say that

> The sync provider may additionally choose to transfer placeholders not matching the pattern.

so.. :shrug: